desktop/get-started.md

```diff
@@ -3,33 +3,28 @@ description: Docker Dashboard
 keywords: Docker Dashboard, manage, containers, gui, dashboard, images, user manual
 title: Sign in and get started
 redirect_from:
-- /docker-for-windows/
-- /docker-for-windows/index/
-- /docker-for-windows/started/
-- /engine/installation/windows/
-- /installation/windows/
-- /win/
-- /windows/
-- /windows/started/
-- /winkit/
-- /winkit/getting-started/
+- /desktop/linux/
 - /desktop/linux/index/
+- /desktop/mac/
+- /desktop/mac/index/
```

Comment:
I wonder if we ever actually had these URLs (with `/index/`); feels like these we added by mistake in the past (but copied multiple times)